data/tlg0062/tlg054/tlg0062.tlg054.perseus-eng3.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Dipsads</title>
+<title xml:lang="grc">Περὶ τῶν διψάδων</title>
+<author>Lucian of Samosata</author>
+<editor role="translator">Kenneth Kilburn</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Gregory Crane</name>
+<name>Lisa Cerrato</name>
+
+</respStmt>
+<funder n="org:NEH">The National Endowment for the Humanities</funder>
+</titleStmt>
+<extent/>
+<publicationStmt>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+    <date type="release" when="2023-06-23">Perseus 5.0</date>
+    <idno type="filename">tlg0062.tlg053.perseus-grc3.xml</idno>
```

Comment:
I'm curious why is this eng3? Isn't this the first English edition Perseus has published? All the rest of the translations have been eng2 before this batch I believe. The first 53 Lucian works were also grc2 and it appears that these are grc3?

Reply:
We already have a grc2 for Jacobitz. For lucian 53-69, eng3 and grc3 are always by the same editor/translator. 
Some day we will do identifiers such as kilburn1959-eng1 (which makes sense). But not today

Reply:
I think I asked because the URN numbers eng3, grc3, in many cases do not necessarily indicate agreement/alignment between a text and translation as it does in this case, the grc2, etc. was previously a way to indicate if a Perseus file had been updated.